alerts/openshift-virtualization-operator/LowReadyVirtControllersCount.md

```diff
@@ -2,68 +2,66 @@
 
-A `virt-controller` device monitors the custom resource definitions (CRDs)
-of a virtual machine instance (VMI) and manages the associated pods. The
-device creates pods for VMIs and manages their lifecycle. The device is
+A `virt-controller` device monitors the custom resource definitions (CRDs) of a
+virtual machine instance (VMI) and manages the associated pods. The device
+create pods for VMIs and manages the lifecycle of the pods. The device is
 critical for cluster-wide virtualization functionality.
 
 ## Impact
 
-This alert indicates that a cluster-level failure might occur. Actions
-related to VM lifecycle management, such as launching a new VMI or
-shutting down an existing VMI, will fail.
+This alert indicates that a cluster-level failure might occur, which would cause
+actions related to VM lifecycle management to fail. This notably includes
+launching a new VMI or shutting down an existing VMI.
 
 ## Diagnosis
 
 1. Set the `NAMESPACE` environment variable:
 
-   ```bash
-   $ export NAMESPACE="$(oc get kubevirt -A \
-     -o custom-columns="":.metadata.namespace)"
-   ```
+  ```bash
+  $ export NAMESPACE="$(oc get kubevirt -A -o custom-columns="":.metadata.namespace)"
+  ```
 
 2. Verify a `virt-controller` device is available:
 
-   ```bash
-   $ oc get deployment -n $NAMESPACE virt-controller \
-     -o jsonpath='{.status.readyReplicas}'
-   ```
+  ```bash
+  $ oc get deployment -n $NAMESPACE virt-controller -o jsonpath='{.status.readyReplicas}'
+  ```
 
 3. Check the status of the `virt-controller` deployment:
 
-   ```bash
-   $ oc -n $NAMESPACE get deploy virt-controller -o yaml
-   ```
+  ```bash
+  $ oc -n $NAMESPACE get deploy virt-controller -o yaml
+  ```
 
-4. Obtain the details of the `virt-controller` deployment to check for
-status conditions, such as crashing pods or failures to pull images:
+4. Obtain the details of the `virt-controller` deployment to check for status
+conditions, such as crashing pods or failures to pull images:
 
-   ```bash
-   $ oc -n $NAMESPACE describe deploy virt-controller
-   ```
+  ```bash
+  $ oc -n $NAMESPACE describe deploy virt-controller
+  ```
 
-5. Check if any problems occurred with the nodes. For example, they might
-be in a `NotReady` state:
+5. Check if any problems occurred with the nodes. For example, they might be in
+a `NotReady` state:
 
-   ```bash
-   $ oc get nodes
-   ```
+  ```bash
+  $ oc get nodes
+  ```
 
 ## Mitigation
 
 This alert can have multiple causes, including the following:
 
-- The cluster has insufficient memory.
-- The nodes are down.
-- The API server is overloaded. For example, the scheduler might be under
-a heavy load and therefore not completely available.
-- There are network issues.
+- Not enough memory on the cluster
+- Nodes are down
+- The API server is overloaded. For example, the scheduler might be under a
+heavy load and therefore not completely available.
+- Networking issues
```

Comment:
This seems like it might be a regression

Reply:
@ousleyp https://github.com/kubevirt/monitoring/pull/243